samples/bootloader/README.rst

```diff
@@ -167,8 +168,8 @@ This would result, for example, in a flash memory layout like the following, whe
 Pre-signed variants
 -------------------
 
-When two slots are present, two images must be built.
-One that is executable from slot 0, and the other one from slot 1.
+When two slots are present, either for the application or for a second-stage bootloader, two images must be built.
+One that is executable from slot 0, and the other one from slot 1, both of which must have :ref:`static partition defined <ug_pm_static>`.
```

Comment:
Maybe we keep the common part in `_ug_bootloader` and link there for information about architecture? User should be aware of the basics anyway.

Reply:
We could also inform there that user must statically define all of the partitions used in a given application configuration.

Reply:
> We could also inform there that user must statically define all of the partitions used in a given application configuration.

Not all partitions. For upgradable bootloaders it goes like this:

B0: should be statically partitioned on release
S0: should be statically partitioned on release
S1: should be statically partitioned on release
mcuboot_primary: can be dymanically partitioned
mcuboot_secondary: can be dymanically partitioned

Reply:
You are right. I slightly oversimplified it here.
Still, the application must be interoperable with the second stage bootloader so that partition maps for the second stage bootloader and application must remain consistent.